apps/meeting/src/containers/MeetingForm/index.tsx

```diff
@@ -142,10 +144,17 @@ const MeetingForm: React.FC = () => {
         label="Join w/o Audio and Video"
         value=""
         checked={isSpectatorModeSelected}
-        onChange={() => (
+        onChange={(): void => (
           setIsSpectatorModeSelected(!isSpectatorModeSelected)
         )}
       />
+      <FormField
+        field={Checkbox}
+        label="Enable Web Audio"
```

Comment:
It would be great if we could add a tooltip component saying "Voice focus works only on enabling Web Audio"

https://aws.github.io/amazon-chime-sdk-component-library-react/?path=/story/ui-components-withtooltip--page

If easy enough, otherwise, we can expand on the label a bit more here.